easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'attr'
+version = '2.4.47'
+
+homepage = 'http://savannah.nongnu.org/projects/attr'
+description = """Commands for Manipulating Filesystem Extended Attributes"""
+
+toolchain = {'name': 'GCC', 'version': '4.6.3'}
```

Comment:
yeah, needed to fix issue with old easyconfig, cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/2699 and https://github.com/hpcugent/easybuild-easyblocks/pull/854